### PR TITLE
Improved account management

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [ ] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Moves the account page to its own screen (instead of nesting under team). Adds new screens to better manage multi-tenancy, including adding/removing new teams and inviting users. 

Note: parts of these new screens will be broken against all non-local Cloud environments until some pending Cloud work is complete.

### CODE REVIEW

File: src/pages/Admin/Teams/NewTeam.vue [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    }
+  },
+  computed: {
+    ...mapGetters('license', ['permissions', 'license', 'planType']),
+    multitenancy() {
+      return this.license?.terms?.tenants > 1
+    }
+  },
+  watch: {},
+  methods: {}
```
Comment (ThatGalNatalie) [inline src/pages/Admin/Teams/NewTeam.vue:21]: can we remove this?

Comment (znicholasbrown) [reply src/pages/Admin/Teams/NewTeam.vue:21]: Sure! Also looks like I have a lot of conflicts to resolve

**Comment by ThatGalNatalie** [comment]
Is it possible to decrease the padding here?
padding

**Comment by ThatGalNatalie** [comment]
Perhaps it's just me but is this supposed to say account two times?
account

**Comment by znicholasbrown** [comment]
@ThatGalNatalie - yeah that's intentional; for enterprise accounts that will be populated with their account name, like this:
Screen Shot 2021-06-23 at 4 53 03 PM

**Review by ThatGalNatalie — APPROVED** [review]
this looks good to me

**Pull request merged**